tests/xtro-sharpie/SimdCheck.cs

```diff
@@ -407,4 +409,4 @@ public override void VisitObjCMethodDecl (ObjCMethodDecl decl, VisitKind visitKi
 			Log.On (framework).Add ($"!unknown-simd-type-in-signature! {method}: the native signature has a simd type ({simd_type}), while the corresponding managed method is using an incorrect (non-simd) type.");
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
hmm

Reply:
New line! Either Rol of you have to agree on one lol

Reply:
I think we agree not to change existing stuff ;-)